website/docs/r/ecs_service.html.markdown

```diff
@@ -113,6 +113,8 @@ The `deployment_controller` configuration block supports the following:
 * `container_name` - (Required) The name of the container to associate with the load balancer (as it appears in a container definition).
 * `container_port` - (Required) The port on the container to associate with the load balancer.
 
+**Note:** With aws provider >=2.22.0 multiple load_balancer blocks are supported. This allows configuration of ECS service support for multiple target groups (https://aws.amazon.com/about-aws/whats-new/2019/07/amazon-ecs-services-now-support-multiple-load-balancer-target-groups/)
```

Comment:
Could you please switch this to use a blue informational block, similar to how we do with the [`for_each` documentation](https://www.terraform.io/docs/configuration/resources.html#for_each-multiple-resource-instances-defined-by-a-map-or-set-of-strings)? Thanks!

```suggestion
-> **Version note:** Multiple `load_balancer` configuration block support was added in Terraform AWS Provider version 2.22.0. This allows configuration of [ECS service support for multiple target groups](https://aws.amazon.com/about-aws/whats-new/2019/07/amazon-ecs-services-now-support-multiple-load-balancer-target-groups/).
```